### PR TITLE
DG Issues

### LINKED ISSUE #197: [PE-D][Tester D] Multiplicity in diagram on DG page 11 is wrong
`Tutorials` multiplicity should be `12` instead of `*`, as there is a fixed number (12) of tutorial sessions, hardcoded into the code.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: shotnothing/ped#15

### LINKED ISSUE #203: [PE-D][Tester B] Notation in Sequence Diagram
There should be a red X for d:DeleteCommand to end the lifeline as well since it's no longer used/referenced afterwards.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: dez2003/ped#4

### LINKED ISSUE #209: [PE-D][Tester D] User stories keep referring to contacts, people and students interchangeably
This can be quite confusing for a developer without prior knowledge of the app's origins as AB3. This happens throughout the user stories.

This is also inconsistent as sometimes, they are referred to as students too, e.g. in DG page 18:

While it is specified in the glossary, I think it should be relabeled to student, nonetheless, by virtue of it being clearer. References to person should be removed.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: shotnothing/ped#20

### LINKED ISSUE #211: [PE-D][Tester D] Placeholder test cases are present on DG page 21
As above.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: shotnothing/ped#21

### LINKED ISSUE #212: [PE-D][Tester A] Class diagram for person
Does Person class contain 12 Tutorial classes, or any number of Tutorials 

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Justincjr/ped#7

### LINKED ISSUE #213: [PE-D][Tester A] No usecase for edit contact
No usecase for edit contact in DG

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Justincjr/ped#8

### LINKED ISSUE #219: [PE-D][Tester A] Define TA in glossary
To ensure that all stakeholders including those that may not be familiar with abbreviations used in NUS have a common understanding

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Justincjr/ped#10

### LINKED ISSUE #220: [PE-D][Tester C] Minor error in DG for manual testing of attendance
Addition of the word 'one' in 'Prerequisites: Ensure one more than one contact is present....."

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: rayray39/ped#9

### LINKED ISSUE #222: [PE-D][Tester C] Mismatch of student ID requirement in DG and error message
In the Developer Guide, under Glossary, the student ID is specified to take the format "eXXXXXXX" with a small letter e. However, when trying to add a new person with an id "e1234567", it is not allowed. Only student id with capital E is allowed. 

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: rayray39/ped#10

### PR DESCRIPTION
**Pull request by DKHB0207**
Made the following changes in the DG:
- Change description for Student ID to be capital "E" instead, resolves #222 
- Typo in editing attendance status, resolves #220 
- Defined TA in glossary, resolves #219 
- Created use case for edit contact, resolves #213 
- Standardise multiplicity of Tutorial Class, resolves #212 , resolves #197 
- Added description and test cases for Saving Data component in Appendix, resolves #211 
- Added Red Cross to indicate end of lifeline for DeleteCommand Object in sequence diagram in Logic Component, resolves #203

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -388,7 +390,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 3d2. List is not empty.
     
         ConTActs will modify all tutorials for every Person.
-         
+
+**UC05: Edit student details**
+
+**MSS**
+
+1.  User requests to list persons.
```
Comment (Domokunx) [inline docs/DeveloperGuide.md:398]: Should we change persons here to students to standardise with other UCs and the rest of the DG?

#209 dependent

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -469,8 +515,27 @@ Prerequisites: Ensure one more than one contact is present. These tests can and
...
+
+   1. Steps to Simulate
+      1. Locate the directory where you saved the `ConTActs.jar` file.
+      2. Locate the `data` folder in this directory.
+      3. Locate the file named `addressbook.json`.
+      4. Delete or temporarily move the data file to simulate a missing file.
+      5. Launch the application once more.
+   2. Expected Behavior
+      1. The application should detect the missing data file.
+      2. A new, empty data file with sample data should be created upon launching the application.
```
Comment (Domokunx) [inline docs/DeveloperGuide.md:528]: empty data file with sample data sounds contradictory. Maybe remove empty?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -469,8 +515,27 @@ Prerequisites: Ensure one more than one contact is present. These tests can and
...
+   1. Steps to Simulate
+      1. Locate the directory where you saved the `ConTActs.jar` file.
+      2. Locate the `data` folder in this directory.
+      3. Locate the file named `addressbook.json`.
+      4. Modify the file to make it invalid JSON by deleting a closing bracket `}`.
+      5. Save the file and close the editor.
+      6. Launch the application once more.
+   2. Expected Behavior
+       1. The application should detect the file is corrupted.
+       2. A new, empty data file should be created upon launching the application.
```
Comment (Domokunx) [inline docs/DeveloperGuide.md:541]: similar

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (sayglenn) [reply docs/DeveloperGuide.md:398]: +1

**Pull request merged**